provider/.github-action.local.template.env

```diff
@@ -0,0 +1,2 @@
+ASA_RESOURCE_SDC_PASSWORD=<FILL_ME>
```

Comment:
What is this?

Reply:
Was thinking to add a template but I guess it is not needed, removed